src/main/java/constants/wordtraceconstants/WordTraceConstants.java

```diff
@@ -4,19 +4,29 @@
  * This class defines constant values used across the WordTrace Application
  */
 public class WordTraceConstants {
+    public static final int COUNT_ZERO = 0;
     public static final String DELIMETER = "\t\n ";
     public static final String EMPTY_STRING = "";
     public static final String ERROR_INSUFFICIENT_ARGUMENTS = "Please provide both the file path and the word to search";
     public static final String ERROR_INVALID_FILE = "File path is incorrect or file doesn't exist. Please provide valid file path";
+    public static final String ERROR_MESSAGE_INVALID_INPUTS = "Invalid Inputs";
+    public static final String ERROR_MESSAGE_WORD_NOT_FOUND = " Word not found!";
```

Comment:
```suggestion
    public static final String INVALID_INPUTS_ERROR_MESSAGE = "Invalid Inputs";
    public static final String WORD_NOT_FOUND_ERROR_MESSAGE = "Search word not found";
```
For invalid inputs pls provide more details.